README.md

```diff
@@ -34,6 +34,16 @@ module "default_vpc" {
   region = "us-west-2"
 }
 ```
+An example that will apply to all regions in an account:
+```hcl
+data "aws_regions" "current" {}
+
+module "default_vpc" {
+  source = "trussworks/destroy-default-vpc/aws"
+  for_each = toset(data.aws_regions.current.names)
+  region = each.value
```

Comment:
It doesn't work for me, assuming we should pass the providers explicitly while working with resources across regions. anything I missed?

Ref: https://www.terraform.io/docs/language/modules/develop/providers.html#passing-providers-explicitly